src/network_analysis.jl

```diff
@@ -192,6 +192,122 @@ function complexstoichmat(::Type{Matrix{Int}}, rn::ReactionSystem, rcs)
     Z
 end
 
+"""
+    laplacianmat(rn::ReactionSystem, pmap=Dict(); sparse=false)
+
+    Return the negative of the graph Laplacian of the reaction network. The ODE system of a chemical reaction network can be factorized as ``\frac{dx}{dt} = Y A_k Φ(x)``, where Y is the [`complexstoichmat`](@ref) and A_k is the negative of the graph Laplacian, and Φ is the [`massactionvector`](@ref). A_k is an n-by-n matrix, where n is the number of complexes, where A_{ij} = k_{ij} if a reaction exists between the two complexes and 0 otherwise. 
```

Comment:
The math here needs to be wrapped appropriately so it displays as math.

I would also put the formula for `A_{ij}` as a standalone math block using cases rather than including it within the text. That latter is easier to read.

Reply:
Similar comment for the other docstrings.

Reply:
What's the right way to wrap math? I was using the double backticks ``N`` since I saw it in other docstrings.

Reply:
Not everything is wrapped here though, for example "A_{ij} = k_{ij}". Look at:

https://github.com/SciML/Catalyst.jl/blob/e3578276888167206dcc006b3a6ae69680718aec/src/network_analysis.jl#L71-L92

and note we needed to use "@doc raw" there to get it to render in the API section correctly.